[2024-10-03T07:23:44.135+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-03T07:23:44.161+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.warehouse.step_1.extract_transform.categories scheduled__2024-10-02T00:00:00+00:00 [queued]>
[2024-10-03T07:23:44.174+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.warehouse.step_1.extract_transform.categories scheduled__2024-10-02T00:00:00+00:00 [queued]>
[2024-10-03T07:23:44.175+0000] {taskinstance.py:2865} INFO - Starting attempt 25 of 25
[2024-10-03T07:23:44.191+0000] {taskinstance.py:2888} INFO - Executing <Task(SparkSubmitOperator): warehouse.step_1.extract_transform.categories> on 2024-10-02 00:00:00+00:00
[2024-10-03T07:23:44.203+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=29427) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-03T07:23:44.205+0000] {standard_task_runner.py:72} INFO - Started process 29429 to run task
[2024-10-03T07:23:44.204+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'warehouse.step_1.extract_transform.categories', 'scheduled__2024-10-02T00:00:00+00:00', '--job-id', '13566', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline/run.py', '--cfg-path', '/tmp/tmppberds6h']
[2024-10-03T07:23:44.208+0000] {standard_task_runner.py:105} INFO - Job 13566: Subtask warehouse.step_1.extract_transform.categories
[2024-10-03T07:23:44.284+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.warehouse.step_1.extract_transform.categories scheduled__2024-10-02T00:00:00+00:00 [running]> on host 8c0d519f07a3
[2024-10-03T07:23:44.412+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='warehouse.step_1.extract_transform.categories' AIRFLOW_CTX_EXECUTION_DATE='2024-10-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='25' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-02T00:00:00+00:00'
[2024-10-03T07:23:44.415+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-03T07:23:44.435+0000] {base.py:84} INFO - Retrieving connection 'spark-conn'
[2024-10-03T07:23:44.436+0000] {spark_submit.py:335} INFO - Spark-Submit cmd: spark-submit --master spark://spark-master:7077 --name arrow-spark dags/etl_pipeline/tasks/warehouse/components/extract_transform.py categories True 2024-10-02
[2024-10-03T07:23:44.496+0000] {spark_submit.py:488} INFO - /home/***/.local/lib/python3.12/site-packages/pyspark/bin/load-spark-env.sh: line 68: ps: command not found
[2024-10-03T07:23:51.836+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:51 INFO SparkContext: Running Spark version 3.5.3
[2024-10-03T07:23:51.839+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:51 INFO SparkContext: OS info Linux, 5.15.153.1-microsoft-standard-WSL2, amd64
[2024-10-03T07:23:51.839+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:51 INFO SparkContext: Java version 17.0.12
[2024-10-03T07:23:51.941+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:51 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-10-03T07:23:52.157+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:52 INFO ResourceUtils: ==============================================================
[2024-10-03T07:23:52.159+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:52 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-10-03T07:23:52.164+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:52 INFO ResourceUtils: ==============================================================
[2024-10-03T07:23:52.166+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:52 INFO SparkContext: Submitted application: Extract Transform - categories
[2024-10-03T07:23:52.213+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:52 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-10-03T07:23:52.236+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:52 INFO ResourceProfile: Limiting resource is cpu
[2024-10-03T07:23:52.238+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:52 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-10-03T07:23:52.358+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:52 INFO SecurityManager: Changing view acls to: ***
[2024-10-03T07:23:52.359+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:52 INFO SecurityManager: Changing modify acls to: ***
[2024-10-03T07:23:52.362+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:52 INFO SecurityManager: Changing view acls groups to:
[2024-10-03T07:23:52.369+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:52 INFO SecurityManager: Changing modify acls groups to:
[2024-10-03T07:23:52.371+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:52 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: ***; groups with view permissions: EMPTY; users with modify permissions: ***; groups with modify permissions: EMPTY
[2024-10-03T07:23:52.812+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:52 INFO Utils: Successfully started service 'sparkDriver' on port 44831.
[2024-10-03T07:23:52.870+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:52 INFO SparkEnv: Registering MapOutputTracker
[2024-10-03T07:23:52.962+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:52 INFO SparkEnv: Registering BlockManagerMaster
[2024-10-03T07:23:53.000+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-10-03T07:23:53.003+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-10-03T07:23:53.009+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-10-03T07:23:53.047+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-ea03d170-baa5-4499-9a6b-72a8434c1ec0
[2024-10-03T07:23:53.080+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-10-03T07:23:53.117+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-10-03T07:23:53.319+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-10-03T07:23:53.433+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-10-03T07:23:53.628+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark-master:7077...
[2024-10-03T07:23:53.715+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO TransportClientFactory: Successfully created connection to spark-master/172.20.0.2:7077 after 53 ms (0 ms spent in bootstraps)
[2024-10-03T07:23:53.886+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO StandaloneSchedulerBackend: Connected to Spark cluster with app ID app-20241003072353-0022
[2024-10-03T07:23:53.888+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20241003072353-0022/0 on worker-20241003061731-172.20.0.9-38803 (172.20.0.9:38803) with 2 core(s)
[2024-10-03T07:23:53.892+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO StandaloneSchedulerBackend: Granted executor ID app-20241003072353-0022/0 on hostPort 172.20.0.9:38803 with 2 core(s), 1024.0 MiB RAM
[2024-10-03T07:23:53.903+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 44381.
[2024-10-03T07:23:53.904+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO NettyBlockTransferService: Server created on 8c0d519f07a3:44381
[2024-10-03T07:23:53.906+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-10-03T07:23:53.916+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 8c0d519f07a3, 44381, None)
[2024-10-03T07:23:53.921+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO BlockManagerMasterEndpoint: Registering block manager 8c0d519f07a3:44381 with 434.4 MiB RAM, BlockManagerId(driver, 8c0d519f07a3, 44381, None)
[2024-10-03T07:23:53.924+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 8c0d519f07a3, 44381, None)
[2024-10-03T07:23:53.927+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 8c0d519f07a3, 44381, None)
[2024-10-03T07:23:53.977+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:53 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20241003072353-0022/0 is now RUNNING
[2024-10-03T07:23:54.489+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:54 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2024-10-03T07:23:56.885+0000] {spark_submit.py:488} INFO - [2024-10-03T07:23:56.883+0000] {base.py:84} INFO - Retrieving connection 'staging_db'
[2024-10-03T07:23:56.907+0000] {spark_submit.py:488} INFO - tidak ada data
[2024-10-03T07:23:56.910+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:56 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-10-03T07:23:56.940+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:56 INFO SparkUI: Stopped Spark web UI at http://8c0d519f07a3:4040
[2024-10-03T07:23:56.952+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:56 INFO StandaloneSchedulerBackend: Shutting down all executors
[2024-10-03T07:23:56.959+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:56 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Asking each executor to shut down
[2024-10-03T07:23:57.009+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:57 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-10-03T07:23:57.028+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:57 INFO MemoryStore: MemoryStore cleared
[2024-10-03T07:23:57.029+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:57 INFO BlockManager: BlockManager stopped
[2024-10-03T07:23:57.039+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:57 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-10-03T07:23:57.044+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:57 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-10-03T07:23:57.063+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:57 INFO SparkContext: Successfully stopped SparkContext
[2024-10-03T07:23:57.548+0000] {spark_submit.py:488} INFO - Traceback (most recent call last):
[2024-10-03T07:23:57.549+0000] {spark_submit.py:488} INFO - File "/opt/***/dags/etl_pipeline/tasks/warehouse/components/extract_transform.py", line 60, in <module>
[2024-10-03T07:23:57.549+0000] {spark_submit.py:488} INFO - ExtractTransform._categories(incremental, date)
[2024-10-03T07:23:57.549+0000] {spark_submit.py:488} INFO - File "/opt/***/dags/etl_pipeline/tasks/warehouse/components/extract_transform.py", line 46, in _categories
[2024-10-03T07:23:57.550+0000] {spark_submit.py:488} INFO - raise e
[2024-10-03T07:23:57.550+0000] {spark_submit.py:488} INFO - File "/opt/***/dags/etl_pipeline/tasks/warehouse/components/extract_transform.py", line 32, in _categories
[2024-10-03T07:23:57.550+0000] {spark_submit.py:488} INFO - raise AirflowSkipException("categories doesn't have new data. Skipped...")
[2024-10-03T07:23:57.550+0000] {spark_submit.py:488} INFO - ***.exceptions.AirflowSkipException: categories doesn't have new data. Skipped...
[2024-10-03T07:23:57.550+0000] {spark_submit.py:488} INFO - [2024-10-03T07:23:57.549+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2024-10-03T07:23:57.967+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:57 INFO ShutdownHookManager: Shutdown hook called
[2024-10-03T07:23:57.968+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:57 INFO ShutdownHookManager: Deleting directory /tmp/spark-e72cf02e-925d-4483-8044-72e1e3527d32/pyspark-cfc0146f-4d28-4152-a647-c1087dd54f0f
[2024-10-03T07:23:57.971+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:57 INFO ShutdownHookManager: Deleting directory /tmp/spark-721ee10a-8a56-4175-90db-b6d9175185f9
[2024-10-03T07:23:57.975+0000] {spark_submit.py:488} INFO - 24/10/03 07:23:57 INFO ShutdownHookManager: Deleting directory /tmp/spark-e72cf02e-925d-4483-8044-72e1e3527d32
[2024-10-03T07:23:58.034+0000] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 157, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 419, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark-master:7077 --name arrow-spark dags/etl_pipeline/tasks/warehouse/components/extract_transform.py categories True 2024-10-02. Error code is: 1.
[2024-10-03T07:23:58.039+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=etl_pipeline, task_id=warehouse.step_1.extract_transform.categories, run_id=scheduled__2024-10-02T00:00:00+00:00, execution_date=20241002T000000, start_date=20241003T072344, end_date=20241003T072358
[2024-10-03T07:23:58.039+0000] {taskinstance.py:1563} INFO - Executing callback at index 0: slack_notifier
[2024-10-03T07:23:58.042+0000] {baseoperator.py:405} WARNING - SlackWebhookOperator.execute cannot be called outside TaskInstance!
[2024-10-03T07:23:58.046+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/slack/hooks/slack_webhook.py:42 UserWarning: You cannot override the default channel (chosen by the user who installed your app), username, or icon when you're using Incoming Webhooks to post messages. Instead, these values will always inherit from the associated Slack app configuration. See: https://api.slack.com/messaging/webhooks#advanced_message_formatting. It is possible to change this values only in Legacy Slack Integration Incoming Webhook: https://api.slack.com/legacy/custom-integrations/messaging/webhooks#legacy-customizations
[2024-10-03T07:23:58.056+0000] {base.py:84} INFO - Retrieving connection 'slack_notifier'
[2024-10-03T07:23:58.510+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-03T07:23:58.511+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 13566 for task warehouse.step_1.extract_transform.categories (Cannot execute: spark-submit --master spark://spark-master:7077 --name arrow-spark dags/etl_pipeline/tasks/warehouse/components/extract_transform.py categories True 2024-10-02. Error code is: 1.; 29429)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 157, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 419, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark-master:7077 --name arrow-spark dags/etl_pipeline/tasks/warehouse/components/extract_transform.py categories True 2024-10-02. Error code is: 1.
[2024-10-03T07:23:58.558+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-10-03T07:23:58.588+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-03T07:23:58.590+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
